raster/src/main/scala/geotrellis/raster/ArrayMultibandTile.scala

```diff
@@ -428,4 +428,21 @@ class ArrayMultibandTile(_bands: Array[Tile]) extends MultibandTile with MacroMu
 
     new ArrayMultibandTile(newBands)
   }
+
+  override def equals(other: Any): Boolean = {
+    if (other == null) {
```

Comment:
Seems like there are two cases: case `that: ArrayMultibandTile` and case `else` - perhaps remove `if (other == null)`?